src/components/notebooks/card.tsx

```diff
@@ -1,32 +1,55 @@
-
 export default function Card(props: Notebook) {
-  const title = props.title || '';
-  const description = props.description || '';
-  // TODO: Fetch notebook 'Last Updated' when implemented on backend
-  const lastUpdated = '';
-  const href = `notebooks/${props.id}/`
+  const title = props.title || "";
+  const description = props.description || "";
+
+  // Formats the received date for UI
+  // Adds '0' to single digit date parts
+  const padNumber = (num: number) => {
+    return num.toLocaleString("en-US", { minimumIntegerDigits: 2 });
+  };
+
+  // Converts the received date to Date object
+  const lastUpdatedDate =
+    new Date(props.lastUpdated) || new Date();
+
+  // Formats the date for UI
+  const lastUpdated =
```

Comment:
I recommend that you use `toLocaleDateString` instead. 
```js
date.toLocaleDateString('en-GB').replaceAll("/", "-"); // DD-MM-YYYY
``` 

In the future, in cases where you need to build string, I recommend using [string interpolation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of concatenation. It's often safer and cleaner.